### PR TITLE
support occupation and enable emoji support for country flag #257

### PR DESCRIPTION
**Pull request by mshd**

### CODE REVIEW

File: src/components/Node/Node.js [hunk 1]
```diff
@@ -97,6 +99,27 @@ export default memo(function Node({
     [node.data.simpleClaims]
   );
 
+  const occupation = useMemo(
+    () =>
+      renderOccupations(occupationByQid(node.data.simpleClaims[OCCUPATION_ID])),
```
Comment (ogroppo) [inline src/components/Node/Node.js:104]: ok, but actually in next I moved these into the formatEntity so are readily available

File: src/components/Node/Node.js [hunk 2]
```diff
@@ -97,6 +99,27 @@ export default memo(function Node({
...
+    [node.data.simpleClaims]
+  );
+
+  function renderOccupations(occ) {
```
Comment (ogroppo) [inline src/components/Node/Node.js:108]: could be a component, but no worries

File: src/components/Node/Node.js [hunk 3]
```diff
@@ -97,6 +99,27 @@ export default memo(function Node({
...
+    console.log(occ);
+    const listItems = occ.map((entry) => (
+      <span title={entry.itemLabel}>{entry.emoji}</span>
+    ));
+    return <div>{listItems}</div>;
+  }
+
+  const getFlag = (c) =>
```
Comment (ogroppo) [inline src/components/Node/Node.js:116]: could be a helper, but no worries, will move in next

File: src/modals/SettingsModal/SettingsModal.js [hunk 4]
```diff
@@ -230,30 +230,14 @@ export default function SettingsModal({ show, hideModal }) {
...
-              >
-                Country flag
-              </Dropdown.Item>
-              <Dropdown.Item
-                active={settings.extraInfo === "religion"}
-                onClick={() => setSetting("extraInfo", "religion")}
-              >
-                religion
-              </Dropdown.Item>
+              {EXTRA_INFO_OPTIONS.map((entry) => (
```
Comment (ogroppo) [inline src/modals/SettingsModal/SettingsModal.js:233]: Yes!

**Review by ogroppo — APPROVED** [review]
some console.logs that can be removed :)

File: src/components/Node/Node.js (hunk 1 above)
Comment (mshd) [reply src/components/Node/Node.js:104]: ok let's move it later

**Pull request merged**